src/components/partials/benefitHubsLinks/index.tsx

```diff
@@ -0,0 +1,29 @@
+// Noting that landing_page is Benefits Hub Landing Page.
+import BenefitsHubLink from '@/components/node/landing_page'
+
+// Partial outputting a list of benefits related to a given page. Typically used
+// on a resource page.
+const BenefitsHubLinks = ({ nodes }) => {
+    if (nodes) {
+        return (
+            <>
+                <section className="vads-u-padding-y--3 vads-u-display--flex vads-u-flex-direction--column vads-u-padding-x--1 large-screen:vads-u-padding-x--0">
+                    <h2 className="vads-u-margin-y--0 vads-u-font-size--h3">
+                        VA benefits
+                    </h2>
+
+                    <ul className="usa-unstyled-list" role="list">
+                        {nodes.map((node, index) => (
+                            <li key={index} className="vads-u-margin-y--2">
```

Comment:
`{index}` is fine for this case. We also have uuidv4 installed as an alternate if needed
`import { v4 as uuidv4 } from 'uuid';`
```
// uuidv4(); // ⇨ '9b1deb4d-3b7d-4bad-9bdd-2b0d7b3dcb6d'
<li key={uuidv4()} className="vads-u-margin-y--2">
```

Reply:
Each node has a uuid; maybe we should use that?